.github/workflows/prerelease.yaml

```diff
@@ -1,7 +1,9 @@
 name: Pre-Release
 
 on:
-  workflow_dispatch:
```

Comment:
Maybe it's safer to leave this in for the time being, for the cases where we might need it anyway? :grimacing: